main_small_batch.py

```diff
@@ -15,23 +15,28 @@
 from pyomo.gdp import Disjunct, Disjunction
 from pyomo.util.infeasible import log_infeasible_constraints
 
-from gdp.dsda.dsda_functions import (external_ref, generate_initialization,
-                                     get_external_information,
-                                     initialize_model,
-                                     solve_complete_external_enumeration,
-                                     solve_subproblem, solve_with_dsda,
-                                     solve_with_gdpopt, solve_with_minlp,
-                                     visualize_dsda)
+from gdp.dsda.dsda_functions import (
+    external_ref,
+    generate_initialization,
+    get_external_information,
+    initialize_model,
+    solve_complete_external_enumeration,
+    solve_subproblem,
+    solve_with_dsda,
+    solve_with_gdpopt,
+    solve_with_minlp,
+    visualize_dsda,
+)
 from gdp.small_batch.gdp_small_batch import build_small_batch
 
 
 def problem_logic_batch(m):
     """
     This function returns the logic expressions to be used in the disjunctive constraints of the D-SDA approach for the small batch problem.
-    
+
     Args:
         m(pyomo.ConcreteModel): Pyomo model of the small batch problem
-    
+
```

Comment:
Missing space behind (pyomo.ConcreteModel)

Reply:
I don't get which is the missing space.

Reply:
between m and (pyomo.ConcreteModel)

Reply:
added the space here.